src/lib.rs

```diff
@@ -275,6 +364,8 @@ extern crate quick_error;
 #[cfg(test)]
 extern crate env_logger;
 extern crate rand;
+#[macro_use]
+extern crate getset;
```

Comment:
I'd like to use this library for most getters and setters (we have several it can replace.) So we could maybe do it now, but I'd remove it later and always use it.

Reply:
Oh, I thought it was a test crate.